### PR TITLE
Update the-day-of-the-exe-is-upon-us.json

### PR DESCRIPTION
**Pull request by amaajemyfren**
Added slides to presentation The day of the exe is upon us.

### CODE REVIEW

File: pycon-us-2014/videos/the-day-of-the-exe-is-upon-us.json [hunk 1]
```diff
@@ -8,6 +8,12 @@
   "language": "eng",
   "quality_notes": "",
   "recorded": "2014-04-13",
+  "related_urls": [
+   {
```
Comment (di) [inline pycon-us-2014/videos/the-day-of-the-exe-is-upon-us.json]:
```suggestion
    {
```

File: pycon-us-2014/videos/the-day-of-the-exe-is-upon-us.json [hunk 2]
```diff
@@ -8,6 +8,12 @@
...
+      "label": "The Day of the EXE Is Upon Us (Slides)",
+      "url": "https://rhodesmill.org/brandon/slides/2014-04-pycon/day-of-the-exe/"
+    }
+    ],
```
Comment (di) [inline pycon-us-2014/videos/the-day-of-the-exe-is-upon-us.json]:
```suggestion
  ],
```

**Comment by amaajemyfren** [comment]
Hey @di,

thanks for the corrections I have made the changes.

Is there some documentation on the format of the page markup? I had to compare with other pages before I could place this pull request.

**Comment by di** [comment]
There's a schema at https://github.com/pyvideo/data/blob/master/.schemas/video.json which may be slightly hard to interpret. A PR adding some documentation about this would be welcomed!

**Pull request merged**